gpkit/tests/t_sub.py

```diff
@@ -203,115 +207,6 @@ def setup(self):
         sol = Top().solve(verbosity=0)
         self.assertAlmostEqual(sol['cost'], 2)
 
-        # add unit test for varsbyname, vector composition
-        # @whoburg: and some for unions and intersections before solving
-
-    # def test_simpleaircraft(self):
```

Comment:
I think we need linked model unit tests -- it could start from the below, or from scratch. But it's a user-facing API so we need to test that what we tell them (e.g. about which variables collide, whether values get overridden, etc) doesn't change.